### ISSUE TITLE
TLS certificate pinning

### ISSUE THREAD

**Issue by luke-jr** [tananaev/traccar-client-android#271]
It would be nice if there was a way to ignore the CAs and lock Traccar to use a specific TLS cert for increased security.

**Comment by tananaev** [comment]
I agree that it would increase security, but I think most users won't be able to set it up. It's too complicated.

**Comment by luke-jr** [comment]
It could be setup by simply displaying the current certificate, and confirming with a Yes/No?

**Comment by tananaev** [comment]
OK, this would make it easier from user point of view, but there are many things that we would have to think about. How to display a certificate, how to reset it etc.

**Comment by stefanb** [comment]
What about just respecting the certificate pinning headers, if they are sent?
